gen3-integration-tests/pages/workspace.py

```diff
@@ -47,61 +42,76 @@ def go_to(self, page: Page):
         """Goes to workspace page and checks if loaded correctly"""
         page.goto(self.BASE_URL)
         page.wait_for_selector(self.READY_CUE, state="visible")
-        screenshot(page, "workspacePage")
+        screenshot(page, "WorkspacePage")
+
+    def assert_page_loaded(self, page: Page):
+        """Checks if workspace page loaded successfully"""
+        page.wait_for_selector(self.READY_CUE, state="visible")
+        screenshot(page, "WorkspacePageLoaded")
 
-    def open_jupyter_workspace(self, page: Page):
-        """Launch a jupyter workspace"""
-        expect(page.locator(self.WORKSPACE_OPTION)).to_be_visible()
-        # find the workspace option for Juptyer WS to launch
-        generic_card = page.locator(self.NOTEBOOK_CARD)
-        expect(generic_card).to_be_visible()
-        # click the launch button from the juptyer workspace card
-        launch_button_xpath = f"{self.NOTEBOOK_CARD}//button[text()='Launch']"
+    def launch_workspace(self, page: Page, name: str = ""):
+        """
+        Launch the workspace with the given name
+        Launches the first available one if `name` is not specified
+        """
+        logger.info("Increasing capacity to speed up the launch")
+        result = run_gen3_command(
+            pytest.namespace, "gen3 ec2 asg-set-capacity jupyter +10"
+        )
+        logger.info(f"Build result: {result}")
+        expect(page.locator(self.WORKSPACE_OPTIONS)).to_be_visible()
+        # launch the specified workspace option if the name is provided
+        logger.info(f"Launching workspace {name}")
+        if name:
+            launch_button_xpath = f"//h3[text()='{name}']/parent::div[@class='workspace-option']/button[text()='Launch']"
+        # launch the first available option if the name is not provided
+        else:
+            launch_button_xpath = (
+                "(//div[@class='workspace-option'])[1]/button[text()='Launch']"
```

Comment:
We can do this in the next PR, but if we can move the launch_button_xpath to init and use it here